misc/ci/emscripten-build-test-ci.ini

```diff
@@ -0,0 +1,53 @@
+[constants]
+cflags = []
+ldflags = ['-v']
+
+[binaries]
+ar = 'emar'
+c = 'emcc'
+cpp = 'em++'
+ranlib = 'emranlib'
+file_packager = '/home/runner/work/taisei/taisei/emsdk/upstream/emscripten/tools/file_packager'
```

Comment:
I'd rather put the toolchain path in constants, and be explicit with all the binaries:

```ini
[constants]
toolchain = '/home/runner/work/taisei/taisei/emsdk/upstream/emscripten'
cflags = []
ldflags = ['-v']

[binaries]
ar = toolchain / 'emar'
c = toolchain / 'emcc'
cpp = toolchain / 'em++'
ranlib = toolchain / 'ranlib'
file_packager = toolchain / 'tools/file_packager'
```

(Note that unquoted `/` is Meson's path-joining operator here)

This setup doesn't rely on emscripten being in `PATH`.

As a further improvement, we can omit the toolchain path from the static crossfile and leverage Meson's layering feature to inject it at CI runtime. This makes it possible to use the crossfile unmodified in other environments.

Pseudo-code:

```sh
cat <<EOF
[constants]
toolchain = '${MY_TOOLCHAIN_PATH}'
EOF > emscripten-path-ephemeral.ini
meson setup \
    --cross-file emscripten-path-ephemeral.ini \
    --cross-file misc/ci/emscripten-build-test-ci.ini \
    source-dir build-dir
```